src/main/java/uk/gov/hmcts/probate/service/ocr/OcrEmailValidator.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.probate.service.ocr;
+
+import static java.lang.String.format;
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.toList;
+
+import uk.gov.hmcts.probate.model.ocr.OCRField;
+
+import java.util.List;
+
+import org.apache.commons.validator.routines.EmailValidator;
+
+public class OcrEmailValidator {
```

Comment:
Prefer to set up this class as a service, like the other validation classes please